pwa-devdocs/src/_drafts/page-builder/how-to-add-your-content-type.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to add your Page Builder content type to PWA Studio
+---
+
+We utilise a special [master format parser](master-format-parser.md) within our Page Builder integration to easily decompose data within the master format for use in a PWA Studio React component.
+
+## Prerequisites 
+
+- Have PWA Studio setup and running in your development environment.
+- Have a working Page Builder component, with the master format implemented.
+- Be able to render and visualise your content type on the existing Luma store front.
+
+## Structure
+As we require some additional steps compared to a normal we have some additional files associated with each content type, in example our `Text` content types component structure looks like the following:
```

Comment:
a normal _what_? getting déjà vu

Reply:
Sorry. This was content that shouldn't have been included in the PR.